index.html

```diff
@@ -15,43 +15,46 @@
       <div class="status-bars">
       </div>
       <div class="app-bar">
-        <div class="toolbar">
+        <div class="toolbar-mobile">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
           <div class="normal-button">
-            <img class="burger" src="assets/burger.svg" alt="Menu icon.">
-            </div>
-            <div class="logo">
-              <span>My Logo</span>
-            </div>
-            <div class="toolbar-container">
-            </div>
+            <img class="burger" src="assets/burger.svg" alt="Menu icon">
           </div>
         </div>
+        <div class="toolbar-desktop">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
+          <nav>
+            <ul class="menu">
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
     </header>
     <main>
-    <section class="headline">
-          <h1 class="primary-text-2">I'm Zermarden <br>
-          Glad to see <br>
-          you!</h1>
-          <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-          <div class="social-media">
-            <div class="supporting-text">
-              <span>LET'S CONNECT!</span>
-            </div>
-          </div>
-          <div class="social-media-icons">
-            <ul class="social-media-icons-list">
-              <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
-              <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
-              <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
-              <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
-              <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
-          </ul>
-          </div>
-        </section>
-        <!--Images should be in Headline section. After that check CSS-->
+      <section class="headline">
```

Comment:
- [ ] [OPTIONAL] Your headline section doesn't have a background color, kindly add a white color to the background so it's similar to the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220604_091326](https://user-images.githubusercontent.com/25227022/171991313-f209b766-ff4d-4fae-a3e3-417c47491e0e.png)|![Screenshot_20220604_091634](https://user-images.githubusercontent.com/25227022/171991318-9e162d11-32b1-4677-a00b-5ed0dbd4d476.png)|